src/main.rs

```diff
@@ -238,8 +240,9 @@ fn download_package( package: &PrebuiltPackage ) -> PathBuf {
     let url = Url::parse( package.url ).unwrap();
     let package_filename = url.path_segments().unwrap().last().unwrap().to_owned();
 
-    let xdg_dirs = xdg::BaseDirectories::with_prefix( "cargo-web" ).unwrap();
-    let unpack_path = xdg_dirs.place_data_file( package.name ).unwrap().join( package.arch );
+    let unpack_path = app_dirs::get_app_dir(app_dirs::AppDataType::UserData, &APP_INFO, package.name )
```

Comment:
1. Missing space after the `(`. (:
2. According to `app_dirs` docs the `get_app_dir` doesn't create missing directories, so probably `app_dir` should be used instead?